build/repo.props

```diff
@@ -8,4 +8,10 @@
   <PropertyGroup>
     <VSTestCLIRunSettings>RunConfiguration.TargetPlatform=x64</VSTestCLIRunSettings>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- These properties are use by the automation that updates dependencies.props -->
+    <LineupPackageId>Internal.AspNetCore.Universe.Lineup</LineupPackageId>
```

Comment:
Does this need to be automatically updated? This is still building against .NET Core 1.1. run upgrade deps would attempt to roll this forward to 2.1 bits